### PR TITLE
[installer] Fix token signing key subpath

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Fixes the path which is mounted onto the pod

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: install/installer/pkg/components/public-api-server/configmap.go
Comment (andrew-farries) [inline install/installer/pkg/components/public-api-server/configmap.go:138]: Curious why we need a subpath mount here at all. If the secret key is also good as a filename then we can just mount the secret directly without a subpath, right?

**Review by andrew-farries — APPROVED** [review]
/hold for question

File: install/installer/pkg/components/public-api-server/configmap.go
Comment (easyCZ) [reply install/installer/pkg/components/public-api-server/configmap.go:138]: Yeah that'd work as well. The difficulty is in sharing the right "secret field key" with the configmap, but that's also "fixable". Let me TAL

Comment (easyCZ) [reply install/installer/pkg/components/public-api-server/configmap.go:138]: Actually, I'll land this PR and investigate in a follow-up, to unblock staging.

**Pull request merged**
